### PR TITLE
Starting Kit changes to run locally

### PR DESCRIPTION
**Pull request by ihsaan-ullah**
### Changes
- removed unused imports from starting kit
- pointer added to conda instructions for users running the starting kit locally
- visualization error removed
- systematics syntax updated
- new package added in requirements.txt
- used test model as sample model (this one works on MAC)

### CODE REVIEW

File: sample_code_submission/boosted_decision_tree.py [hunk 1]
```diff
@@ -1,90 +1,74 @@
...
+
 
 class BoostedDecisionTree:
     """
     This class implements a boosted decision tree classifier.
 
     Attributes:
-        * model (XGBClassifier): The underlying XGBoost classifier model.
-        * scaler (StandardScaler): The scaler used to normalize the input data.
+        model (XGBClassifier): The underlying XGBoost classifier model.
```
Comment (Ragansu) [inline sample_code_submission/boosted_decision_tree.py:14]: why did you change this ? its causing issues in MACOS ?
the `* ` was added for sphinx documentation

File: sample_code_submission/model.py [hunk 2]
```diff
@@ -38,34 +69,40 @@ def __init__(self, get_train_set=None, systematics=None):
...
-
-        self.systematics = systematics
-
         self.get_train_set = get_train_set
-
+        self.systematics = systematics
         self.re_train = True
 
-        from boosted_decision_tree import BoostedDecisionTree
+        if XGBOOST:
```
Comment (Ragansu) [inline sample_code_submission/model.py:76]: So you brought added the 3 models from test_model ?

File: sample_code_submission/statistical_analysis.py [hunk 3]
```diff
@@ -1,459 +1,83 @@
...
 import pickle
-from iminuit import Minuit
-import matplotlib.pyplot as plt
 
 path.append("../")
 path.append("../ingestion_program")
 
 
-class StatisticalAnalysis:
+def compute_mu(score, weight, saved_info):
```
Comment (Ragansu) [inline sample_code_submission/statistical_analysis.py:10]: This basicly undoes all of yuleis work.

**Review by Ragansu — CHANGES_REQUESTED** [review]

**Comment by Ragansu** [comment]
Over all we cant remove a lot of things. The NLL model should still be there.

File: sample_code_submission/boosted_decision_tree.py (hunk 1 above)
Comment (ihsaan-ullah) [reply sample_code_submission/boosted_decision_tree.py:14]: I just took the code from codabench submission. Maybe that was old

File: sample_code_submission/model.py (hunk 2 above)
Comment (ihsaan-ullah) [reply sample_code_submission/model.py:76]: I haven't changed anything in this file. It is the test model on codabench

File: sample_code_submission/statistical_analysis.py (hunk 3 above)
Comment (ihsaan-ullah) [reply sample_code_submission/statistical_analysis.py:10]: nothing changed by me here. Maybe I took a wrong test model

**Comment by ihsaan-ullah** [comment]
Screenshot 2024-08-19 at 3 21 54 PM

This is the model i have used

**Comment by ihsaan-ullah** [comment]
> Over all we cant remove a lot of things. The NLL model should still be there.

Should I put back the removed model? I don't understand what needs to be done

**Comment by Ragansu** [comment]
for the test model here is my suggestion. 
* Dont touch the sample submission instead rename the `sameple_code_submission` (yulei model) as example_submission  
* Rename `test_model` to `sample_code_submission`. 
* use `git mv` so that it doesn't show up as lines changed in PR

**Review by Ragansu — APPROVED** [review]
looks good
